src/prompts/generateApiSpecFromPrompt.ts

```diff
@@ -40,4 +40,16 @@ Examples defined in the components section must be used.
 Server variables must be defined and valid and there must be no unused variables.
 Callbacks should not be defined within a callback
 Servers should not be defined in a webhook.
-Callbacks should not be defined in a webhook.
+Callbacks should not be defined in a webhook.`;
+
+
+export default function (context: any): string {
+    if (!(context.vars.hasOwnProperty('ruleContent'))) {
```

Comment:
Why adding this? It is changing original logic.

Reply:
updated